requirements/edx-sandbox/py38.txt

```diff
@@ -0,0 +1,97 @@
+#
```

Comment:
It looks like this is a copy rather than a symlink, was that intentional? I think it makes more sense to make this a symlink.